[20405] Meaningful labels (accessibility)

mk84 opened the pull request:
Part of: https://community.openproject.org/work_packages/20405/activity

I also added a new translation key: `label_create_work_package: "Create new work package"`, which needs to be translated.

Some issues are not solved yet – see https://community.openproject.org/work_packages/20405/activity#activity-11

mk84 committed:
Fix wp view mode buttons labels for screenreaders

mk84 committed:
Add label on create new wp button for screenreaders

mk84 committed:
Use singular on wp create button

mk84 committed:
Add aria-haspopup and aria-expanded attributes to topmenu

mk84 committed:
Add aria-haspopup on wp create button

TeatroIO commented:
I've prepared a stage to preview changes. <a href='http://mk84-20405-meaningful-labels.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/' target='_blank'>Open stage</a> or <a href='http://mk84-20405-meaningful-labels.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com//teatro.log' target='_blank'>view logs</a>.

oliverguenther committed:
Restore accessible-by-keyboard template

This fixes the broken specs due to:

1. Switch to button in accessible-by-keyboard
2. Changed label for '+ Work Packages'
